pkg/noderesourcetopology/least_numa.go

```diff
@@ -228,3 +230,13 @@ func checkResourcesFit(identifier string, qos v1.PodQOSClass, resources v1.Resou
 
 	return true
 }
+
+func getNumaIDInNumaNodes(numaNodes NUMANodeList, nodeIndex int) int {
+	numaID := 0
```

Comment:
is this correct? if we don't match the node we return still a valid node. We should either return -1 or an error.